### PR TITLE
Documented a way of building MyGUI

### PR DESCRIPTION
**Pull request by QuestionableM**
-----
<a href="https://stackblitz.com/~/github.com/scrap-mods/docs/tree/Misc-BuildingMyGUI">Review PR in StackBlitz</a> _Submitted with [StackBlitz](https://stackblitz.com/~/github.com/scrap-mods/docs/tree/Misc-BuildingMyGUI)._

### CODE REVIEW

File: docs/misc/docs/mygui-building.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Configuring MyGUI with CMake
+1. Set **MYGUI_RENDERSYSTEM** to **4** (OpenGL)
+2. Set **FREETYPE_LIBRARY** as the path to **freetype.lib** file
+3. Set **FREETYPE_INCLUDE_DIRS** to the include directory of freetype
+4. Set **SDL2_INCLUDE_DIR** to the include directory of SDL2
+5. Set **SDL2_LIBRARY** as the path to **SDL2.lib** file
+6. Set **SDL2_IMAGE_INCLUDE_DIR** to the include directory of SDL2 Image
+7. Set **SDL2_IMAGE_LIBRARY** as the path to **SDL2_image.lib** file
+
+## Compile MyGUI and have fun
```
Comment (TechnologicNick) [inline docs/misc/docs/mygui-building.mdx]: Having fun is not allowed.

File: docs/misc/docs/mygui-building.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: Building MyGUI
+---
+
+# Building MyGUI
+
+Building MyGUI was kinda painful, so it's time to document the entire process so it's not lost to time.
+
+# Getting MyGUI
```
Comment (TechnologicNick) [inline docs/misc/docs/mygui-building.mdx]: For Markdown you should structure your headings as a hierarchy, so # is the title and ## a subsection and ### a subsection.

File: docs/misc/docs/mygui-building.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+3. Download and compile [**FreeType**](https://github.com/freetype/freetype)
+
+# Configuring MyGUI with CMake
+1. Set **MYGUI_RENDERSYSTEM** to **4** (OpenGL)
+2. Set **FREETYPE_LIBRARY** as the path to **freetype.lib** file
+3. Set **FREETYPE_INCLUDE_DIRS** to the include directory of freetype
+4. Set **SDL2_INCLUDE_DIR** to the include directory of SDL2
+5. Set **SDL2_LIBRARY** as the path to **SDL2.lib** file
+6. Set **SDL2_IMAGE_INCLUDE_DIR** to the include directory of SDL2 Image
+7. Set **SDL2_IMAGE_LIBRARY** as the path to **SDL2_image.lib** file
```
Comment (TechnologicNick) [inline docs/misc/docs/mygui-building.mdx]: Maybe use \`inline code blocks\` for these?

**Review by TechnologicNick — CHANGES_REQUESTED** [review]
Some minor things. Also maybe change the name of the file to `building-mygui` to match the title?

File: docs/misc/docs/mygui-building.mdx (hunk 1 above)
Comment (QuestionableM) [reply docs/misc/docs/mygui-building.mdx]: 🗿

**Review by TechnologicNick — APPROVED** [review]
After complying with picky nick it now looks good to me

**Pull request merged**